styles.css

```diff
@@ -1,3 +1,71 @@
+
+.modal {
+    display: none;
+    position: fixed;
+    z-index: 1;
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    background-color: rgba(0, 0, 0, 0.19);
```

Comment:
Stick to hex values if you're using them elsewhere. You should also be able to specify opacity with hex.